app/src/main/res/layout/change_address_button.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Button xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should we create a generic outlined button-layout file, and use that whenever we need one of these?

I know this breaks the norm for the repository, but I think it could be wise to move towards that kind of structure, as we currently have a lot of layouts that basically are just `TextView`s or `Button`s with the same style that only really differ in what `android:text` they have. Something to think about, for sure 🙂

Reply:
Yes agreed, feels a bit out of scope for this PR though. Compose will be so nice in this regard 💯